Scripts/Runtime/Logging/.CSProject/UnityLogListener.cs

```diff
@@ -13,11 +15,96 @@ namespace Anvil.Unity.Logging
     [DefaultLogListener]
     public sealed class UnityLogListener : ILogListener, UnityEngine.ILogHandler
     {
+        /// <summary>
+        /// Place on a method to make the <see cref="UnityLogListener"> use the next 
+        /// call up the stack for the log caller context.
+        /// </summary>
+        [AttributeUsage(AttributeTargets.Method)]
+        public class ExcludeAttribute : Attribute { }
+
         /// <summary>
         /// The context delivered when a context can't be resolved
         /// </summary>
         private sealed class UnknownContext { }
 
+        private readonly struct LogMessage
+        {
+            public readonly string Message;
+            public readonly LogLevel LogLevel;
+            public readonly bool IsFromBurstedContext;
+
+            public LogMessage(string message, LogLevel logLevel, bool isFromBurstedContext)
+            {
+                Message = message;
+                LogLevel = logLevel;
+                IsFromBurstedContext = isFromBurstedContext;
+            }
+        }
+
+        // Pumps events off the main thread to indicate that pending logs
+        // should be processed.
+        // We want to process the queue as often as possible (within reason) so we can handle
+        // the logs as close to the time they were emitted as possible.
+        [DefaultExecutionOrder(int.MaxValue)]   // Execute last in each phase
+        private sealed class PendingLogPump : MonoBehaviour
```

Comment:
A custom pump was created because the Logging project doesn't reference the greater anvil-unity-core framework and lives within it as a DLL.

This would be a terribly inefficient use case for `UpdateHandle` anyway!